app/assets/javascripts/angular/controllers/my_requests.js.erb

```diff
@@ -2,4 +2,8 @@ app.controller('MyRequestsCtrl', ['$scope', '$http', '$location', '$auth', funct
 
   var url = "<%= ENV['URL'] %>" || "http://localhost:3000";
 
+  $http.get(url + '/requests').success(function(response){
+    console.log(response);
```

Comment:
Instead of console logging, why don't you try to use [bootstrap alerts](http://getbootstrap.com/components/#alerts) to inform the user?